org.eclipse.wildwebdeveloper/pom.xml

```diff
@@ -114,14 +115,14 @@
 								<arg>install</arg>
 								<arg>--no-bin-links</arg>
 								<arg>--ignore-scripts</arg>
-								<arg>./node-debug2-1.33.0.tgz</arg>
+								<arg>./node-debug2-1.42.1.tgz</arg>
 								<arg>typescript@3.7.2</arg>
 								<arg>typescript-language-server@0.4.0</arg>
 								<arg>typescript-lit-html-plugin@0.9.0</arg>
 								<arg>typescript-plugin-css-modules@2.1.0</arg>
 								<arg>./vscode-css-languageserver-1.0.0.tgz</arg>
 								<arg>./vscode-html-languageserver-1.0.0.tgz</arg>
-								<arg>vscode-json-languageserver@1.2.2</arg>
+								<arg>./vscode-json-languageserver-1.2.2.tgz</arg>
```

Comment:
So the json ls on npm.js is different from the one in vscode?

Reply:
yes, and the one on npmjs is currently bugged: https://github.com/microsoft/vscode/issues/90634
Actually, I think it's often best ot just pick what's directly in VSCode rather than on npm; it seems like it's better tested.